### ISSUE TITLE
Support additional config options in torrc template

### ISSUE THREAD

**Issue by conorsch** [systemli/ansible-role-hidden-service#8]
Magnificent role, thank you for developing and publishing this. Would you entertain a PR that exposes more options in the torrc file, such as `Sandbox`? Options specific to relays are likely out of scope for this role (see the excellent [nensu.tor-relayor](https://github.com/nusenu/ansible-relayor)), but many options affect hidden services and are not configurable via this role.

In practice, admins can use `lineinfile` tasks to maintain custom options in the torrc, but that breaks idempotence at the playbook level. I'm happy to submit a PR with short list of configurable options and associated default vars. The defaults will match those of the tor service itself, naturally.

**Comment by xshadow** [comment]
Thank you. Every option that affects hidden services are very welcome

**Comment by 0x46616c6b** [comment]
I prefer a flexible configuration like:

**defaults/main.yml**

```
hidden_services_configuration:
    SocksPort: 9050
    SocksPolicy: "reject *"
```

In template you would only iterate over the dict:

**templates/torrc.j2**

```
{% for key, value in hidden_services_configuration.iteritems() %}
{{ key }} {{ value }}
{% endfor %}
```

**Comment by conorsch** [comment]
Great, thanks for the quick reply. @0x46616c6b I see you're using that iteration format in the template already, so I'll stick to it. Keeps things simple and trivially extensible.

**Comment by 0x46616c6b** [comment]
I discussed on friday the option with dict vs. explicit definition. both approaches has benefits. with the dict you can simple add new options, but the drawback: you need to overwrite the whole config in every case you only need to update one line (e.g. the port). So in this case the options are limited and we should parameterize every option in torrc instead of the dict.

**Comment by t2d** [comment]
As @0x46616c6b says both approaches have it's benefits and drawbacks. ATM I'm not really sure which is the best way to go in general. So I merged #9 to support @conorsch use case. Refactoring could be done at a later point.
